public/typescript/ui/component/startAuthentication.ts

```diff
@@ -0,0 +1,21 @@
+import { Observable } from 'rxjs';
```

Comment:
This isn't a React component, please move this out of the `ui/component/` folder.